code/game/objects/items/weapons/cards_ids_syndicate.dm

```diff
@@ -205,3 +205,15 @@
 
 /datum/card_state/dd_SortValue()
 	return name
+
+/obj/item/card/id/syndicate/ert
+	name = "operative id card"
+	desc = "A specialized ID for the appointed operatives of the high council."
+	registered_name = "Agent"
+	assignment = "Agent"
+	icon_state = "hc_all-access"
+	group = "golden"
+
+/obj/item/card/id/secert/New()
+	access = get_all_station_access()
```

Comment:
each ERT agent was giving different accesses that aline with their job not AA

Reply:
An access console *is* present in the CC area if we decide that a particular agent needs to have less access, but we should generally trust that ERTs(Who are often sent to assist in emergencies) can handle having emergency access imo

Reply:
We dont trust agents tho, agents are people